### DISCUSSION TITLE
Get Pull Request URL From Commit Message in BitBucket?

### DISCUSSION THREAD

**Discussion by vovkanaz** [orhun/git-cliff#527] [Q&A]
**Is your feature request related to a problem? Please describe.**
We Have this Commit message inside Bitbucket:

MERGED
Merged in [BRANCH NAME](branch_url) [pull request #50](https://bitbucket.org/project_name/repo_url/pull-requests/id) commit_message

**Describe the solution you'd like**
Does Cliff support any ability for ingesting git PR/MR url inside CHANGELOG MD?

**Describe alternatives you've considered**

We've found [LINK](https://github.com/orhun/git-cliff/blob/main/examples/cocogitto.toml#L60 ) as this example, but still do not have clear info on how to align this with our case.  It seems like we should parse our PR commit message to be able to ingest it inside CHANGELOG.md. But still haven't had success using such an approach.

**Additional context**

1. How to ingest the result of this preprocessed message into the template? Or we should use post_processor? But a question is the same how to use it inside the template after processing?
2. Also another question may be some thoughts about the automation process related to the automatic TAG creation using Bitbucket pipelines.

**Comment by orhun** [comment]
Hey!

> MERGED
> Merged in [BRANCH NAME](https://github.com/orhun/git-cliff/discussions/branch_url) [pull request #50](https://bitbucket.org/project_name/repo_url/pull-requests/id) commit_message

I didn't quite get this format. Can you share the output of `git show` or `git log` command?

> How to ingest the result of this preprocessed message into the template? Or we should use post_processor? But a question is the same how to use it inside the template after processing?

Can you share an example of the changelog format that you want to achieve?

> Also another question may be some thoughts about the automation process related to the automatic TAG creation using Bitbucket pipelines.

Integration of `git-cliff` should be similar to any automation provider - you can simply install it and use `--bump`/`--bumped-version` flags for the next version.
